unidist/cli/backends/dask/runner.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2021 Modin authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""``DaskRunner`` class functionality."""
+
+import os
+import warnings
+
+from unidist.cli.base.runner import BackendRunner
+from unidist.cli.base.utils import Defaults, validate_num_cpus
+from unidist.core.base.common import BackendName
+
+
+class DaskRunner(BackendRunner):
+    """
+    An implementation of unidist ``BackendRunner`` for Dask backend.
+
+    Parameters
+    ----------
+    **cli_kwargs : dict
+        Keyword arguments supported by unidist CLI.
+    """
+
+    def __init__(self, **cli_kwargs):
+        self.backend = BackendName.DASK
```

Comment:
This is wrong because we set backend here and then override its value to default backend value in base class. I think one initialization of the parameter is enough.

Reply:
everywhere

Reply:
Agree. Init in `BackendRunner` is enough